datetimeparser/enums.py

```diff
@@ -1,41 +1,44 @@
     FALL_BEGIN = Constant('fall begin', ['fall', 'begin of fall', 'begin of the fall', 'autumn begin', 'autumn',
-                                         'begin of autumn', 'begin of the autumn'])
-    SUMMER_END = Constant('summer end', ['end of summer', 'end of the summer'])
-    WINTER_END = Constant('winter end', ['end of winter', 'end of the winter'])
-    SPRING_END = Constant('spring end', ['end of spring', 'end of the spring'])
-    FALL_END = Constant('fall end', ['end of fall', 'end of the fall', 'autumn end', 'end of autumn', 'end of the autumn'])
+                                         'begin of autumn', 'begin of the autumn'], time_value=lambda year_time: datetime(year=year_time, month=9, day=1))
+    SUMMER_END = Constant('summer end', ['end of summer', 'end of the summer'], time_value=lambda year_time: datetime(year=year_time, month=8, day=31, hour=23, minute=59, second=59))
+    WINTER_END = Constant('winter end', ['end of winter', 'end of the winter'], time_value=lambda year_time: datetime(year=year_time, month=2, day=days_feb(year_time), hour=23, minute=59, second=59))
+    SPRING_END = Constant('spring end', ['end of spring', 'end of the spring'], time_value=lambda year_time: datetime(year=year_time, month=5, day=31, hour=23, minute=59, second=59))
+    FALL_END = Constant('fall end', ['end of fall', 'end of the fall', 'autumn end', 'end of autumn', 'end of the autumn'], time_value=lambda year_time: datetime(year=year_time, month=11, day=30, hour=23, minute=59, second=59))
 
     MORNING = Constant('morning', ['at morning', 'in the next morning', 'in the morning'])
     EVENING = Constant('evening', ['at evening', 'in the next evening', 'in the evening'])
     LUNCHTIME = Constant('lunchtime', ['lunch'])
 
     BEGIN_AOC = Constant('aoc begin', ['aoc', 'next aoc', 'begin of aoc', 'begin of the aoc', 'advent of code begin',
                                        'advent of code', 'next advent of code', 'begin of advent of code',
-                                       'begin of the advent of code'])
+                                       'begin of the advent of code'], time_value=lambda year_time: datetime(year=year_time, month=12, day=1, hour=6))
     END_AOC = Constant('aoc end', ['end of aoc', 'end of the aoc', 'advent of code end', 'end of advent of code',
-                                   'end of the advent of code'])
+                                   'end of the advent of code'], time_value=lambda year_time: datetime(year=year_time, month=12, day=26, hour=6))
 
-    END_OF_YEAR = Constant('end of year', ['the end of year', 'the end of the year', 'end of the year'])
-    BEGIN_OF_YEAR = Constant('begin of year', ['the begin of year', 'the begin of the year', 'begin of the year'])
+    END_OF_YEAR = Constant('end of year', ['the end of year', 'the end of the year', 'end of the year'], time_value=lambda year_time: datetime(year=year_time, month=12, day=31, hour=23, minute=59, second=59))
```

Comment:
Line too long, look at other lines where they've been already splitted onto multiple lines and do the same